DigitisingSC/DHSC-public-website
# Feature/dhsc 53  front end accessibility

# Test

- check if Storybook has accessibility support:
- check if content pages have accessibility checker on the bottom right corner when you test them in preview mode
